ArduPlane/radio.cpp

```diff
@@ -402,7 +402,7 @@ bool Plane::rc_throttle_value_ok(void) const
  */
 bool Plane::rc_failsafe_active(void) const
 {
-    if (!rc_throttle_value_ok()) {
+    if (!rc_throttle_value_ok() || (ThrFailsafe(g.throttle_fs_enabled.get()) == ThrFailsafe::Disabled)) {
```

Comment:
this is a bit unintuitive for someone reading the code because I think it means that the RC failsafe is active when it's actually been disabled.

Reply:
i think this is already covered in Plane::rc_throttle_value_ok() implementation?

Reply:
it is not....rc_throttle_value_ok()  early returns, yes, but ALWAYS when throttle failsafe is disabled, even when failsafe flag is still set ....rc_failsafe_active will return true and never return false after RC sis restored

Reply:
@rmackay9 and that is exactly the problem...this function returns true even if rc failsafe has been disabled otherwise...I guess it could be handled somehow in the rc_throttle_okay function...but I don't think it really belongs there either